spec/event_spec.rb

```diff
@@ -96,6 +96,18 @@
     end
   end
 
+  describe '.parse' do
```

Comment:
I don't think it harms anything to leave it in. Should be fast enough and verifies that we set up the `Parser#component` properly.